### DISCUSSION TITLE
Paste code on the TIC80 web interface?

### DISCUSSION THREAD

**Discussion by fabbricius** [nesbox/TIC-80#1610] [Q&A] [answered]
Hello,
is it possible to paste code on the web interface ? I want someone I know to test my program before releasing it, so I created a web account on tic80.com and I wanted to paste the code and sending him the id and password. It's not really a game, it's more a tool that I did for him in order to learn the TIC-80 programming. I designed the code in the Atom editor (I have a PRO version in order to launch tic80 directly from the Atom editor).

**Comment by nesbox** [comment, marked as answer]
Hi,
Unfortunately, the browser doesn't have access to the OS clipboard, you can only paste to the textedit control.

**Reply by ChildishGiant** [reply]
It [should be technically possible](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard) although I have no clue how WASM will get in the way.

**Reply by nesbox** [reply]
Ok, let's wait until it's added to the WASM version of the SDL2 lib.
Thank you.
